test/fluture.test.js

```diff
@@ -1404,6 +1580,29 @@ describe('Compliance', function(){
       ));
     });
 
+    describe('ChainRec', () => {
```

Comment:
No. It uses the Static-Land compatible functions rather than Fantasy-Land compatible methods inside its law-tests.